### PR TITLE
Base collection and intrinsics documentation

### PR DESCRIPTION
**Pull request by laytan**

### CODE REVIEW

File: resources/odin-doc.json [hunk 1]
```diff
@@ -1,5 +1,19 @@
 {
     "collections": {
+        "base": {
+            "name": "base",
+            "source_url": "https://github.com/odin-lang/Odin/tree/master/base",
+            "base_url": "/base",
+            "root_path": "$ODIN_ROOTbase",
```
Comment (flysand7) [inline resources/odin-doc.json:7]: This doesn't look right to me, is that correct?

Comment (laytan) [reply resources/odin-doc.json:7]: It is yeah, same thing with the other collections below, `$ODIN_ROOT` is search and replaces with the path with a slash suffix.

**Comment by laytan** [comment]
I updated my fork with the changes from this PR for easier reviewing: https://new-pkg-poc.laytan.dev/

**Comment by laytan** [comment]
Ok ready for merge

**Pull request closed without merging**